cluster/store/store.go

```diff
@@ -248,7 +246,9 @@ func (st *Store) Open(ctx context.Context) (err error) {
 		"last_log_applied_index", st.initialLastAppliedIndex,
 		"last_snapshot_index", lastSnapshotIndex)
 
-	go st.onLeaderFound(st.migrationTimeout)
+	// There's no hard limit on the migration, so it should take as long as necessary.
+	// However, we believe that 1 day should be more than sufficient.
+	go st.onLeaderFound(time.Hour * 24)
```

Comment:
just a suggestion: you can make a const out of `time.Hour * 24` just to make it more visible

Reply:
I was thinking to make it an environment variable just in case later on! What do you think?